readthedocs/settings/base.py

```diff
@@ -101,7 +101,9 @@ def SHOW_DEBUG_TOOLBAR(self):
     def SWITCH_PRODUCTION_DOMAIN(self):
         if self.RTD_EXT_THEME_ENABLED:
```

Comment:
```suggestion
        # TODO: DELETE THIS MARCH 12  
        if self.RTD_EXT_THEME_ENABLED:
```

😆

Reply:
Yeah, we will need to grep the code and remove all of these 👍🏼